bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -111,7 +111,8 @@ void FileStorage::write(
 
 bool FileStorage::isNewStorage() { return (objectsMetadata_ == nullptr); }
 
-bool FileStorage::initMaxSizeOfObjects(ObjectDesc *metadataObjectsArray, uint32_t metadataObjectsArrayLength) {
+bool FileStorage::initMaxSizeOfObjects(const std::map<uint32_t, ObjectDesc> &metadataObjectsArray,
```

Comment:
As it's a map now, can we remove metadataObjectsArrayLength

Reply:
agreed, but I would like to change the existing API as less as possible